modules/ROOT/pages/spring-data-neo4j.adoc

```diff
@@ -1,318 +1,478 @@
 = Spring Data Neo4j
+:doctype: book
```

Comment:
not sure if this breaks things. @Mogztter ?

Reply:
Not sure either and unless I'm missing something I don't think we need it here?

Reply:
FWIW, I basically cherry-pick @meistermeier SDN/RX commit from a few months ago and replaced any references to RX with SDN 6.
Will remove, thanks for spotting this.